tests/unit/project_loader/grammar/test_to_statement.py

```diff
@@ -39,14 +39,14 @@ class ToStatementGrammarTestCase(GrammarBaseTestCase):
             'body': ['foo'],
             'else_bodies': [],
             'target_arch': None,
-            'expected_packages': {'foo:amd64'}
```

Comment:
why are these changes in the commit they are in, I lack context

Reply:
Because this commit removed the `ToStatement`'s arch-patching, using the base class's implementation instead.